### PR TITLE
Make followees modal UI more compact

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Make followees modal UI more compact.

# Changes

- review followees spacing and style
- display subtitle only when collapsible is opened
- do not display "currently following" if no following
- bump gix-cmp to be able to specifiy a css variable to force overflow to scroll for content

# Screenshots

Capture d’écran 2023-04-17 à 11 50 12
Capture d’écran 2023-04-17 à 11 50 22
Capture d’écran 2023-04-17 à 11 50 25
Capture d’écran 2023-04-17 à 11 50 29
Capture d’écran 2023-04-17 à 11 50 31
Capture d’écran 2023-04-17 à 11 50 33
Capture d’écran 2023-04-17 à 11 50 39
Capture d’écran 2023-04-17 à 11 50 41
Capture d’écran 2023-04-17 à 11 50 14

### CODE REVIEW

File: frontend/package-lock.json
Comment (lmuntaner) [inline frontend/package-lock.json:1]: Why did you need to upgrade gix-components?

**Review by lmuntaner — APPROVED** [review]
Only one comment, already approving

File: frontend/package-lock.json
Comment (peterpeterparker) [reply frontend/package-lock.json:1]: See `Changes`

> bump gix-cmp to be able to specifiy a css variable to force overflow to scroll for content

There was no css variable to do so before so had to add a css variable in gix cmp.

Comment (lmuntaner) [reply frontend/package-lock.json:1]: Oh, sorry, I focused on the screenshots and didn't read the description well. My bad.

Comment (peterpeterparker) [reply frontend/package-lock.json:1]: No worries, thanks for the review
